### PR TITLE
Fix Vote Skip Command Bug.

### PR DESCRIPTION
**Pull request by ezBot03**
There was a problem which made the bot bug out if the members in a VC was only 1, with the bot, but this bug has now been fixed, please merge this to fix the bug.

### CODE REVIEW

**Comment by ezBot03** [comment]
Resolved Conflicts.

**Comment by ezBot03** [comment]
Also added an Emoji Fix, so now the title emoji's in the help embed will work, they were broken before.

**Review by l1ving — APPROVED** [review]

File: commands/uptime.js [hunk 1]
```diff
@@ -1,38 +1,29 @@
 const Discord = require("discord.js");
+const fs = require("graceful-fs");
+const exec = require('child_process').exec;
 
-module.exports.run = async (client, message, args) => {
+module.exports.run = async (client, message, args) => {
```
Comment (humboldt123) [inline commands/uptime.js]: Replace server uptime with bot uptime.

Comment (ezBot03) [reply commands/uptime.js]: Resolved i think now

**Comment by humboldt123** [comment]
Thanks!

**Comment by ezBot03** [comment]
ty

**Comment by ezBot03** [comment]
np xd

**Pull request merged**